guacamole/src/main/webapp/app/client/controllers/clientController.js

```diff
@@ -154,25 +154,41 @@ angular.module('client').controller('clientController', ['$scope', '$routeParams
         callback: RECONNECT_ACTION.callback,
         remaining: 15
     };
-    
-    // Hide menu by default
-    $scope.menuShown = false;
 
-    // Use physical keyboard by default
-    $scope.inputMethod = 'none';
+    /**
+     * Menu-specific properties.
+     */
+    $scope.menu = {
+
+        /**
+         * Whether the menu is currently shown.
+         *
+         * @type Boolean
+         */
+        shown : false,
+
+        /**
+         * The currently selected input method. This may be either 'none',
```

Comment:
Oops - you're right. I said "none" twice. Why does GitHub never include enough context... The second "none" is cut off in the diff excerpt.